coderd/database/dbauthz/setup_test.go

```diff
@@ -52,7 +55,11 @@ type MethodTestSuite struct {
 // SetupSuite sets up the suite by creating a map of all methods on querier
 // and setting their count to 0.
 func (s *MethodTestSuite) SetupSuite() {
-	az := dbauthz.New(nil, nil, slog.Make())
+	ctrl := gomock.NewController(s.T())
+	mockStore := dbmock.NewMockStore(ctrl)
```

Comment:
@Emyrk I think it makes sense to use the mock here instead of `nil`, as it will allow us  to immediately see if we accidentally call methods of the underlying `database.Store`.

Reply:
Ah I see though, for the wrappers method.